python_modules/libraries/dagster-teradata/dagster_teradata/resources.py

```diff
@@ -0,0 +1,464 @@
+                    teradata.execute_query(
+                        "DROP DATABASE IF EXISTS MY_DATABASE"
+                    )
+        """
+        check.str_param(sql, "sql")
+
+        with self.get_connection() as conn:
+            with closing(conn.cursor()) as cursor:
+                self.log.info("Executing query: " + sql)
+                cursor.execute(sql)
+                if fetch_results:
+                    if single_result_row:
+                        return self._single_result_row_handler(cursor)
+                    else:
+                        return cursor.fetchall()
+
+    @public
+    def execute_queries(
+        self,
+        sql_queries: Sequence[str],
+        fetch_results: bool = False,
+    ) -> Optional[Sequence[Any]]:
+        """Execute multiple queries in Teradata.
+
+        Args:
+            sql_queries (str): List of queries to be executed in series
+
+        Examples:
+            .. code-block:: python
+
+                @op
+                def create_fresh_database(teradata: TeradataResource):
+                    queries = ["DROP DATABASE MY_DATABASE", "CREATE DATABASE MY_DATABASE"]
+                    teradata.execute_queries(
+                        sql_queries=queries
+                    )
+
+        """
+        check.sequence_param(sql_queries, "sql_queries", of_type=str)
+
+        results: List[Any] = []
+        with self.get_connection() as conn:
+            with closing(conn.cursor()) as cursor:
+                for sql in sql_queries:
+                    self.log.info("Executing query: " + sql)
+                    cursor.execute(sql)
+                    if fetch_results:
+                        results = results.append(cursor.fetchall())  # type: ignore
+                        return results
+
+    @public
+    def s3_to_teradata(
+        self,
+        s3: S3Resource,
+        s3_source_key: str,
+        teradata_table: str,
+        public_bucket: bool = False,
+        teradata_authorization_name: str = "",
+    ):
+        """Loads CSV, JSON and Parquet format data from Amazon S3 to Teradata.
+
+        Args:
+            :param s3_source_key: The URI format specifying the location of the S3 bucket.(templated)
+                The URI format is /s3/YOUR-BUCKET.s3.amazonaws.com/YOUR-BUCKET-NAME.
+                Refer to
+                https://docs.teradata.com/search/documents?query=native+object+store&sort=last_update&virtual-field=title_only&content-lang=en-US
+            :param public_bucket: Specifies whether the provided S3 bucket is public. If the bucket is public,
+                it means that anyone can access the objects within it via a URL without requiring authentication.
+                If the bucket is private and authentication is not provided, the operator will throw an exception.
+            :param teradata_table: The name of the teradata table to which the data is transferred.(templated)
```

Comment:
can you rearrange the fields in comment as per method parameters order?

Reply:
I will